docroot/profiles/custom/cgov_site/modules/custom/cgov_promo/cgov_promo.info.yml

```diff
@@ -5,6 +5,7 @@ description: 'Promo item functionality for CGov Digital Platform Sites'
 core_version_requirement: '^9.4 || ^10'
 dependencies:
   - 'cgov_core:cgov_core'
+  - 'cgov_home_landing:cgov_home_landing'
```

Comment:
This is not correct. I am sure this is what features dumped out, but what this means is that something from cgov_home_landing is in the wrong place.

Reply:
Basically whatever is going on here is going to be a circular reference. cgov_home_landing requires cgov_promo and cgov_promo requires cgov_home_landing.

Is `field.storage.node.field_external_link_display` (I leave it to the reader to figure out the full name) in cgov_home_landing?

Reply:
From Slack from Dev to me:
> On Friday, while I was on call with you, Paul and Andy, he had an issue while reinstalling Drupal with an "unmet dependency" error on his field.field file. While I was on call, we found that his file was put in cgov_home_landing by features when we wanted it to be in cgov_promo. We identified that error and moved the file to cgov_promo on Friday, and I didn't realize that cgov_promo.info.yml got updated too. Paul's fix to revert that file was correct as we moved the field.field file to cgov_promo. I checked his branch, features is happy with the changes.